### ISSUE TITLE
can't play glb file

### ISSUE THREAD

**Issue by GooM33Z** [google/model-viewer#4863]
### Description

We have a glb file that the viewer can't open and we don't get any error messages. maybe you can see what's wrong with the file?
Another model files goes well, as you can see in our demo page:
https://jk93d.bizdorado.com/model/idc_powair_ham/6

We uploaded the wrong glb file here:
https://drive.google.com/file/d/1BRqGMKoV1JNCKtlYmduMrej_5IlnIV8H/view?usp=sharing

#### Live Demo

https://glitch.com/edit/#!/model-viewer

### Version

- model-viewer: v3.5.0

### Browser Affected

not relevant

### OS

not relevant

### AR

not relevant

**Comment by elalish** [comment]
Yeah, it loads fine, but there doesn't appear to be any visible geometry. I would guess this non-ratified extension is to blame: `FB_ngon_encoding`. Try exporting without that - I don't know of any glTF viewers that handle non-triangles.
